### PR TITLE
Bug 1210304 - David's comments related to clock were fixed..

### PR DESCRIPTION
**Pull request by ruvsmirnov**
Clock functionality was transfered (installing localization and handling timeformatchange event) from "app.js" file to "clock.js" file.

### CODE REVIEW

**Comment by ruvsmirnov** [comment]
@harman-red-square/harman-reviewers

File: apps/verticalhome/js/clock.js [hunk 1]
```diff
@@ -36,6 +36,74 @@ function Clock() {
...
+
+  };
+
+  /**
+   * We need to do some refreshing thing after l10n is ready.
+   *
+   * @memberof LockScreen
+   * @this {LockScreen}
+   */
+  this.l10nInit = function ls_l10nInit() {
```
Comment (egorlevichev) [inline apps/verticalhome/js/clock.js]: as I can see, named functions in this file have prefix 'cl_' meaning 'clock'. Please use this prefix for newly added functions

File: apps/verticalhome/js/clock.js [hunk 2]
```diff
@@ -45,18 +113,18 @@ function Clock() {
    *        tick, should accept a date object as its only argument.
    * @memberOf Clock
    */
-  this.start = function cl_start(refresh) {
+  this.start = function cl_start() {
```
Comment (egorlevichev) [inline apps/verticalhome/js/clock.js:126]: as soon as you are changing function's signature, you shall update it's description. By the way, is it intended to stop passing callback as a parameter?

**Comment by egorlevichev** [comment]
please fix couple of inline comments. also it would be nice to rephrase commit message to make it 'more english' :)

File: apps/verticalhome/js/clock.js (hunk 2 above)
Comment (ruvsmirnov) [inline apps/verticalhome/js/clock.js:126]: Fixed

File: apps/verticalhome/js/clock.js (hunk 1 above)
Comment (ruvsmirnov) [inline apps/verticalhome/js/clock.js]: Fixed

File: apps/verticalhome/js/clock.js [hunk 3]
```diff
@@ -36,27 +36,95 @@ function Clock() {
...
+
+        break;
+    };
+
+  };
+
+  /**
+   * We need to do some refreshing thing after l10n is ready.
+   *
+   * @memberof LockScreen
```
Comment (egorlevichev) [inline apps/verticalhome/js/clock.js]: and still LockScreen smell in your commit. BTW why for this particular function you specified '@memberof' tag, and for others no? Probably it'll be better to align documentation either delete here, or add for remaining functions.

**Comment by egorlevichev** [comment]
following comment from @AlekoNN in #16, @TatianaMigacheva please merge this one too, if for these PRs we don't need to wait for remaining 2 labels to be merged

**Pull request merged**